src/Serialization/Rings.jl

```diff
@@ -239,6 +239,14 @@ function load_internal(s::DeserializerState,
     return matrix(entries_ring, mat)
 end
 
+function load_internal_with_parent(s::DeserializerState,
```

Comment:
Can we also get a test for this new function, to verify it now works as desired? 

Also, does this PR resolve issue #1719? If so, please add `resolves #1719` to the PR description, then it'll automatically close that issue once it gets merged :-)

Reply:
The matrix tests cover these lines now, I don't know why the warning is still here?